pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "ansys-rocky-core"
-version = "0.1.dev0"
+version = "0.2.dev0"
```

Comment:
I never noticed the `.dev0` here

Reply:
we only removed the dev on the release branch